certbot-dns-cloudflare/certbot_dns_cloudflare/__init__.py

```diff
@@ -3,6 +3,10 @@
 completing a ``dns-01`` challenge (`~acme.challenges.DNS01`) by creating, and
 subsequently removing, TXT records using the Cloudflare API.
 
+.. note::
+   The plugin is not installed by default. It can be installed by heading to
+   `certbot.eff.org <https://certbot.eff.org/instructions#wildcard/>`_, choosing your system and
```

Comment:
Just one stray trailing slash here, but otherwise this looks great, thank you.

```suggestion
   `certbot.eff.org <https://certbot.eff.org/instructions#wildcard>`_, choosing your system and
```